site/app/templates/officeHoursQueue/QueueStatus.twig

```diff
@@ -0,0 +1,117 @@
+    {% if viewer.getQueueNumberAheadOfYou(viewer.getCurrentQueueCode()) - 1 == 1 %}
+      <p>There is currently
+        {{viewer.getQueueNumberAheadOfYou(viewer.getCurrentQueueCode()) - 1 }}
+        person ahead of you in the queue</p>
+    {% else %}
+      <p>There are currently
+        {{viewer.getQueueNumberAheadOfYou(viewer.getCurrentQueueCode()) - 1 }}
+        people ahead of you in the queue</p>
+    {% endif %}
+    <p>You are currently in:
+      <b>{{viewer.removeUnderScores(viewer.getCurrentQueueCode())}}</b>
+    </p>
+
+    <p>
+      {% if viewer.firstTimeInQueueThisWeek(viewer.getCurrentQueueLastHelped()) %}
+        Because you have not yet been helped through this queue during the last week, you may be prioritized.
+      {% elseif viewer.firstTimeInQueueToday(viewer.getCurrentQueueLastHelped()) %}
+        Because you have not yet been helped through this queue today, you may be prioritized.
+      {% else %}
+        However, because you have been helped through this queue today, you may be de-prioritized.
+      {% endif %}
+    </p>
+    <p>
+      {% set num_ahead_week = viewer.numberAheadInQueueThisWeek() %}
+      {% set num_ahead_today = viewer.numberAheadInQueueToday() %}
+      {% if viewer.firstTimeInQueueThisWeek(viewer.getCurrentQueueLastHelped()) %}
+        There
+        {{viewer.plurality(num_ahead_week, 'is', 'are')}}
+        currently
+        {{num_ahead_week}}
+        {{viewer.plurality(num_ahead_week, 'person', 'people')}}
+        ahead of you in the queue who
+        {{viewer.plurality(num_ahead_week, 'has', 'have')}}
+        not yet been helped in the last week, and
+        {{num_ahead_today}}
+        additional
+        {{viewer.plurality(num_ahead_today, 'person', 'people')}}
+        ahead of you in the queue who have not yet been helped today.
+      {% elseif viewer.firstTimeInQueueToday(viewer.getCurrentQueueLastHelped()) %}
+        There
+        {{viewer.plurality(num_ahead_week, 'is', 'are')}}
+        currently
+        {{num_ahead_week}}
+        {{viewer.plurality(num_ahead_week, 'person', 'people')}}
+        in the queue who
+        {{viewer.plurality(num_ahead_week, 'has', 'have')}}
+        not been helped in the last week (they may be prioritized over you), and
+        {{num_ahead_today}}
+        {{viewer.plurality(num_ahead_today, 'person', 'people')}}
+        ahead of you in the queue who have not yet been helped today.
+      {% else %}
+        There
+        {{viewer.plurality(num_ahead_week, 'is', 'are')}}
+        currently
+        {{num_ahead_week}}
+        {{viewer.plurality(num_ahead_week, 'person', 'people')}}
+        in the queue who
+        {{viewer.plurality(num_ahead_week, 'has', 'have')}}
+        not yet been helped in the last week, and
+        {{num_ahead_today}}
+        additional
+        {{viewer.plurality(num_ahead_today, 'person', 'people')}}
+        in the queue who have not yet been helped today.
+      {% endif %}
```

Comment:
There's only a couple strings that differ between each branch, so let's use the if/elseif/else to set the strings that are unique, and then just output the block once.

```
    {% set queue_text = 'in the queue who' %}
    {% set position_text = 'not yet been helped in the last week, and' %}
    {% set end_text = 'ahead of you in the queue who have not yet been helped today' %}
    {% if viewer.firstTimeInQueueThisWeek(viewer.getCurrentQueueLastHelped()) %}
        {% set queue_text = 'ahead of you in the queue who' }
    {% elseif not viewer.firstTimeInQueueToday(viewer.getCurrentQueueLastHelped()) %}
        {% set end_text = 'in the queue who have not yet been helped today' %}
    {% endif %}
    There {{viewer.plurality(num_ahead_week, 'is', 'are')}} currently
    {{num_ahead_week}} {{viewer.plurality(num_ahead_week, 'person', 'people')}}
    {{ queue_text }} {{viewer.plurality(num_ahead_week, 'has', 'have')}}
    {{ position_text }} {{num_ahead_today}} additional
    {{viewer.plurality(num_ahead_today, 'person', 'people')}} {{ end_text }}.
```